database/migrations/2017_09_20_062736_add_safety_agreement_column.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSafetyAgreementColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->timestamp('accept_safety_agreement')->nullable();
```

Comment:
Not that it makes too much difference but for general sanity can you add an `->after('column') to this and put it before the timestamps in this table? I don't have the rest of the schema on hand at the moment to give you a column to put it after.